src/History.tsx

```diff
@@ -4,7 +4,7 @@ import { Timestamp } from './Timestamp.tsx';
 import { MessageGroup, HistoryActions } from './App.tsx';
 import { HistoryMessage } from './HistoryMessage.tsx';
 
-interface Props {
+export interface Props {
```

Comment:
Does this actually help anything? At some point I tried this and it didn't seem to make a difference.

Ditto with the other props you exported below.